RandomData.Api/Helpers/SwashbuckleHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
+
+namespace RandomData.Api.Helpers
+{
+    public static class SwashbuckleServiceHelper
+    {
+        public static IServiceCollection AddSwashbuckle(this IServiceCollection serviceCollection) =>
+            serviceCollection.AddSwaggerGen(setup =>
+            {
+                setup.SwaggerDoc("v1", new OpenApiInfo()
+                {
+                    Title = "RandomData",
+                    Version = "v1"
+                });
+                var xmlFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
```

Comment:
I think you can move these 3 lines (20-22) to separate function which describe this process, for example: "AddXmlComments".